template/README.md

```diff
@@ -228,9 +228,11 @@ import './Button.css'; // Tell Webpack that Button.js uses these styles
 class Button extends Component {
   render() {
     // You can use them as regular CSS styles
-    return <div className="Button" />;
```

Comment:
It’s not correct: in the new code, `Button` renders to `Button` which will cause infinite recursion.
What you probably want to change is to make the first example return `<div className="Button" />` instead of vague `// ...`.